### ISSUE TITLE
Version 3.0 testing feedback & known issues

### ISSUE THREAD

**Issue by Arxareon** [Arxareon/MovementSpeed#14]
Any general feedback, recommendations or info on issues found during the testing of version 3.0 is welcome here!
Please refer to the known issues list in the release notes first.

[Version 3.0 (Test B) release can be found here!](https://github.com/Arxareon/MovementSpeed/releases/tag/3.0_Test_B)

**Comment by mario-64** [comment]
After today's 11.0.2 game update, MovementSpeed is no longer working. I'm using 3.0 Test B. The addons are active in the addon list but /movespeed doesn't work and there is no speed display or output whatsoever.

**Comment by OttoDeFe** [comment]
There seems to be a bunch of calls that need to be updated to the C_AddOns versions in WidgetTools. I was hunting them down, but ran out of time this evening.

https://warcraft.wiki.gg/wiki/Patch_11.0.2/API_changes#Deprecated_API

**Comment by Arxareon** [comment]
Test C should resolve this issue, thanks for the reports!

**Comment by mario-64** [comment]
Is Test C available?

**Comment by Arxareon** [comment]
> Is Test C available?

Right, I forgot to mention. No, it's not available yet. I'll try to get the fix for context menus in quickly before I drop it, **today**. If I can't, I'll still release it to solve the breaking issues and keep the matter of context menus as a known issue for now.

**Comment by mario-64** [comment]
Confirmed with Test C I have the speed frame again. Thank you!

**Comment by Arxareon** [comment]
Test D is out, adding back support for context menus.
https://github.com/Arxareon/MovementSpeed/releases/tag/3.0_Test_D

**I'd really appreciate the testing of settings and reports on any issues found (besides Profiles not yet working properly). After I solve known issues and any other bugs found via additional testing, 3.0 can finally be green-lit as a stable release.**

**Comment by Arxareon** [comment]
Test E is now out, readding support for Classic versions of the game & more: [https://github.com/Arxareon/MovementSpeed/releases/tag/3.0_Test_E](https://github.com/Arxareon/MovementSpeed/releases/tag/3.0_Test_E)

**Comment by mario-64** [comment]
> Test E is now out, readding support for Classic versions of the game & more: https://github.com/Arxareon/MovementSpeed/releases/tag/3.0_Test_E

The same files work for both retail and classic?

**Comment by Arxareon** [comment]
> The same files work for both retail and classic?

@mario-64 Yes, [MovementSpeed-3.0_Test_E.zip](https://github.com/Arxareon/MovementSpeed/releases/download/3.0_Test_E/MovementSpeed-3.0_Test_E.zip) can be used for both.
`Main.lua` will be loaded for retail, `Main_Classic.lua` will be loaded for Classic versions, but it's all included in the same install package. That is the intention. If it doesn't work for any version of WoW, please let me know. If that's the case, there is an error I missed and I'll try to fix it asap!

**Comment by mario-64** [comment]
> > The same files work for both retail and classic?
> 
> @mario-64 Yes, [MovementSpeed-3.0_Test_E.zip](https://github.com/Arxareon/MovementSpeed/releases/download/3.0_Test_E/MovementSpeed-3.0_Test_E.zip) can be used for both. `Main.lua` will be loaded for retail, `Main_Classic.lua` will be loaded for Classic versions, but it's all included in the same install package. That is the intention. If it doesn't work for any version of WoW, please let me know. If that's the case, there is an error I missed and I'll try to fix it asap!

Cool thank you

**Comment by Arxareon** [comment]
The latest beta version is here: [Movement Speed 3.0 Test F (Release candidate)](https://github.com/Arxareon/MovementSpeed/releases/tag/3.0_Test_F)

I'll wait for some testing and do a full release in a couple of days / next week (after making any quick changes/fixes as needed).

**Comment by Arxareon** [comment]
The final release is out (with the fist, latest hotfix): [Movement Speed 3.0.1](https://github.com/Arxareon/MovementSpeed/releases/tag/3.0.1)

### Known issues:
 - Vanilla Classic doesn't support the new addon loading feature `AllowLoadGameType` yet. Until I or Blizzard will fix it, please revert to an earlier version. Thank you fo undestanding!

_Any testing is most appreciated!_ ♥
